### PR TITLE
Add Notes on Remote WID

### PR DESCRIPTION
**Pull request by bryandam**
WID does not support remote connections.

Therefore, the WSUS maintenance actions requiring remote DB connection (indexes and removing obsolete updates) will fail and there's nothing the user can do to fix it.

(Not sure why that last change got included ... some end-line thing?)

### CODE REVIEW

**Comment by PRMerger8** [comment]
@bryandam : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger12** [comment]
@mestew : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by mestew** [comment]
#sign-off

File: memdocs/configmgr/sum/deploy-use/software-updates-maintenance.md [hunk 1]
```diff
@@ -130,7 +130,8 @@ When the WSUS database is on a remote SQL Server, you might need to add permissi
 - The `CONNECT SQL` server permission must be granted to the account. For more information, see [GRANT Server Permissions (Transact-SQL)](/sql/t-sql/statements/grant-server-permissions-transact-sql).
 
 > [!NOTE]  
->  If the WSUS database is on a remote SQL Server using a non-default port, then indexes might not be added. You can create a [server alias using SQL Server Configuration Manager](/sql/database-engine/configure-windows/create-or-delete-a-server-alias-for-use-by-a-client) for this scenario. Once the alias is added and Configuration Manager can make a connection to the WSUS database, indexes will be added.
+> - If the WSUS database is on a remote SQL Server using a non-default port, then indexes might not be added. You can create a [server alias using SQL Server Configuration Manager](/sql/database-engine/configure-windows/create-or-delete-a-server-alias-for-use-by-a-client) for this scenario. Once the alias is added and Configuration Manager can make a connection to the WSUS database, indexes will be added.
+> - If the Software Update Point is remote to the site server and is using a Windows Internal Database then the indexes will not be added.
```
Comment (ShannonLeavitt) [inline memdocs/configmgr/sum/deploy-use/software-updates-maintenance.md:134]:
```suggestion
> - If the Software Update Point is remote to the site server and is using a Windows Internal Database, then the indexes will not be added.
```
Editorial change

File: memdocs/configmgr/sum/deploy-use/software-updates-maintenance.md [hunk 2]
```diff
@@ -149,6 +150,9 @@ When the WSUS database is on a remote SQL Server, the site server's computer acc
 - The `db_datareader` and `db_datawriter` fixed database roles. For more information, see [Database-Level Roles](/sql/relational-databases/security/authentication-access/database-level-roles#fixed-database-roles).
 - The `CONNECT SQL` server permission must be granted to the site server's computer account. For more information, see [GRANT Server Permissions (Transact-SQL)](/sql/t-sql/statements/grant-server-permissions-transact-sql).
 
+> [!NOTE]  
+> If the Software Update Point is remote to the site server and is using a Windows Internal Database then obsolete updates will not be removed.
```
Comment (ShannonLeavitt) [inline memdocs/configmgr/sum/deploy-use/software-updates-maintenance.md:154]:
```suggestion
> If the Software Update Point is remote to the site server and is using a Windows Internal Database, then obsolete updates will not be removed.
```
Editorial change

**Comment by PRMerger14** [comment]
@ShannonLeavitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by bryand-recastsoftware** [comment]
I may, in actual fact, have received an F on my comma usage quiz in grade 3.  I will go to my grave both overusing them and underusing them.

**Pull request merged**